views/applications/one.html

```diff
@@ -0,0 +1,26 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{config.actions.edit|capitalize}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+  </div>
+  {% include "applications/_form.html" %}
+{% endblock %}
```

Comment:
View (and story) is missing the display of the Client ID and Client Secret, as well as buttons for revoking and resetting the tokens.